packages/api/seeders/20210225144642-demo-user.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
```

Comment:
lets also delete these empty migrations

Reply:
Should i delete all migrations?

Reply:
Done

Reply:
I think we fixed the issue with the migrations being deleted but we still want to delete these could of empty seed files.